consensus/ibft/ibft.go

```diff
@@ -793,7 +797,7 @@ func (i *Ibft) runAcceptState() { // start new round
 			}
 
 			// calculate how much time do we have to wait to mine the block
-			delay := time.Until(time.Unix(int64(i.state.block.Header.Timestamp), 0))
+			delay := time.Until(time.UnixMilli(int64(i.state.block.Header.Timestamp)))
```

Comment:
same as well

Reply:
Resolved backward compatibility with two custom functions in consensus/utils.go that do the same thing 631515404cf370d1fc015938ea35da52ff9dbaaf

Reply:
You're linking the wrong commit but the change is there :) @ZeljkoBenovic  Resolving comment